### ISSUE TITLE
sab 2.1.0--develop -- traceback on startup

### ISSUE THREAD

**Issue by thezoggy** [sabnzbd/sabnzbd#921]
i have my main instance (windows binary) then went to startup my dev instance.. forgot to do the `-f` option..  thankfully it didnt nuke out my main instance but did discover it crashed on startup:
```
C:\Program Files (x86)\sabnzbd_git>python -O SABnzbd.py --new
Traceback (most recent call last):
  File "SABnzbd.py", line 1696, in <module>
    main()
  File "SABnzbd.py", line 1056, in main
    newport = find_free_port(browserhost, port)
  File "SABnzbd.py", line 674, in find_free_port
    raise IOError
IOError
```

**Comment by thezoggy** [comment]
i'll submit a patch for this tomorrow, going to make it exit sab correctly during this scenario (panic)..

```
C:\Program Files (x86)\sabnzbd_git>python -O SABnzbd.py -f C:\test\sabnzbd.ini --new

Fatal error:
  Initialization Error
Unable to bind to 192.168.0.2:8080
```

**Comment by sanderjo** [comment]
Is this related to https://github.com/sabnzbd/sabnzbd/issues/875 ?

**Comment by thezoggy** [comment]
@sanderjo its due to https://github.com/sabnzbd/sabnzbd/commit/c9f765813ce85628f91f32a0c044aefa17200102

**Comment by Safihre** [comment]
@thezoggy I was working on this before my holiday, because I also saw I broke it. 
But then I found that the whole panic page system doesn't work very well. It seems very old code with bits in there that are not used anymore and others that might never have worked at all. 
I also noticed that if you have 'Launch browser on startup' turned off, it never shows any panic messages which is a bit confusing.
